arduino/OpenMRN.h

```diff
@@ -340,6 +371,18 @@ class OpenMRN : private Executable
         stack_->executor()->loop_once();
     }
 
+#if defined(ESP32)
+    static void openmrn_background_task(void *param)
+    {
+        OpenMRN *openmrn = reinterpret_cast<OpenMRN *>(param)
+        while(true)
+        {
+            openmrn->loop();
+            vTaskDelay(OPENMRN_TASK_TICK_DELAY);
```

Comment:
is there a better method to call here? this thread will be donated to the executor but if it doesn't yield in some manner it will cause the other threads to be starved. That is primarily why there is the vTaskDelay.